website/docs/reference/directives.md

```diff
@@ -90,8 +90,8 @@ Add Java options
 
 Add Java properties
 
-`//> using javaProp_ `_key=value_
-`//> using javaProp_ `_key_
+`//> using javaProp `_key=value_
```

Comment:
Actually, the docs under `website/docs/reference` aren't written by hand, but instead generated automatically with a script. This is why your change is causing the `reference-doc` target fail on CI.

To actually fix this typo, you'd have to modify the definition of [`UsingJavaPropsDirectiveHandler.usageMd`](https://github.com/VirtusLab/scala-cli/blob/fee54928c39d2e9b8a37e9f3fa42b86c203dcafa/modules/directives/src/main/scala/scala/build/preprocessing/directives/UsingJavaPropsDirectiveHandler.scala#L11) and then run:
```
./mill -i generate-reference-doc.run
```